indicators/NO_GJEN_001/R/NO_GJEN_001.qmd

```diff
@@ -1448,35 +1505,32 @@ vGrid1 <- vaatmarkIndexGrid %>%
   geom_sf(data=regions, fill=NA, size=0.5) +
   scale_fill_gradientn(colors=brewer.pal(10, 'Spectral'),
                        limits = c(0,1),
-                       name = 'Index') +
+                       name = 'Tilstandsverdi') +
   theme_void() +
-  theme(legend.position = c(0.7, 0.4),
-        plot.title = element_text(size=10)) +
-  ggtitle('A) Våtmark gjengroing condition')
+  theme(legend.position = c(0.8, 0.4)) +
+  ggtitle('A) Våtmark')
 
 vGrid2 <- aapneIndexGrid %>%
   ggplot() +
   geom_sf(aes(fill=index), color=NA) +
   geom_sf(data=regions, fill=NA, size=0.5) +
   scale_fill_gradientn(colors=brewer.pal(10, 'Spectral'),
                        limits = c(0,1),
-                       name = 'Index') +
+                       name = 'Tilstandsverdi') +
   theme_void() +
-  theme(legend.position = c(0.7, 0.4),
-        plot.title = element_text(size=10)) +
-  ggtitle('B) Naturlig åpne condition')
+  theme(legend.position = c(0.8, 0.4)) +
+  ggtitle('B) Naturlig åpen')
 
 vGrid3 <- semiIndexGrid %>%
   ggplot() +
   geom_sf(aes(fill=index), color=NA) +
   geom_sf(data=regions, fill=NA, size=0.5) +
   scale_fill_gradientn(colors=brewer.pal(10, 'Spectral'),
                        limits = c(0,1),
-                       name = 'Index') +
+                       name = 'Tilstandsverdi') +
   theme_void() +
-  theme(legend.position = c(0.7, 0.4),
-        plot.title = element_text(size=10)) +
-  ggtitle('C) Semi-naturlig condition')
+  theme(legend.position = c(0.8, 0.4)) +
+  ggtitle('C) Semi-naturlig')
 
 
 
```

Comment:
IN fig-regional-level-enc-forest caption, add note that indicator values are not final, and serve only as prrof of concept for the indicator workflow